### ISSUE TITLE
rake task NameError: uninitialized constant Kitchen::Driver::Docker::Tempfile

### ISSUE THREAD

**Issue by robertlabrie** [test-kitchen/kitchen-docker#206]
I use the Rake embedded with chef for various reasons. When running the rake task I get this error:

```
NameError: uninitialized constant Kitchen::Driver::Docker::Tempfile
/opt/chef/embedded/lib/ruby/gems/2.1.0/gems/kitchen-docker-2.4.0/lib/kitchen/driver/docker.rb:279:in `build_image'

```
Adding:
`require 'tempfile'` to the top of docker.rb resolved it for me

Some info about me:
```

mylaptop> /opt/chef/embedded/bin/rake -V
rake, version 11.1.2
mylaptop> /opt/chef/embedded/bin/ruby -v
ruby 2.1.8p440 (2015-12-16 revision 53160) [x86_64-darwin13.0]
mylaptop> /opt/chef/embedded/bin/chef-client -v
Chef: 12.9.38
```

**Comment by gregsymons** [comment]
This is fixed by #209

**Comment by coderanger** [comment]
Thanky :)
